bigtable/README.rst

```diff
@@ -1,53 +1,89 @@
+3. `Enable the Cloud Bigtable API.`_
+4. `Setup Authentication.`_
 
-For more information on setting up your Python development environment,
-such as installing ``pip`` and ``virtualenv`` on your system, please refer
-to `Python Development Environment Setup Guide`_ for Google Cloud Platform.
+.. _Select or create a Cloud Platform project.: https://console.cloud.google.com/project
+.. _Enable billing for your project.: https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project
+.. _Enable the Cloud Bigtable API.:  https://cloud.google.com/bigtable
+.. _Setup Authentication.: https://googlecloudplatform.github.io/google-cloud-python/latest/core/auth.html
 
-.. _Python Development Environment Setup Guide: https://cloud.google.com/python/setup
+Installation
+~~~~~~~~~~~~
 
-Authentication
---------------
+Install this library in a `virtualenv`_ using pip. `virtualenv`_ is a tool to
+create isolated Python environments. The basic problem it addresses is one of
+dependencies and versions, and indirectly permissions.
 
-With ``google-cloud-python`` we try to make authentication as painless as
-possible. Check out the `Authentication section`_ in our documentation to
-learn more. You may also find the `authentication document`_ shared by all
-the ``google-cloud-*`` libraries to be helpful.
+With `virtualenv`_, it's possible to install this library without needing system
+install permissions, and without clashing with the installed system
+dependencies.
 
-.. _Authentication section: https://google-cloud-python.readthedocs.io/en/latest/core/auth.html
-.. _authentication document: https://github.com/GoogleCloudPlatform/google-cloud-common/tree/master/authentication
+.. _`virtualenv`: https://virtualenv.pypa.io/en/latest/
 
-Using the API
--------------
 
-Cloud `Bigtable`_  is Google's NoSQL Big Data database service. It's the same
-database that powers many core Google services, including Search,
-Analytics, Maps, and Gmail.
+Mac/Linux
+^^^^^^^^^
 
-.. _Bigtable: https://cloud.google.com/bigtable/docs/
+.. code-block:: console
 
-See the ``google-cloud-python`` API Bigtable `Documentation`_ to learn
-how to manage your data in Bigtable tables.
+    pip install virtualenv
+    virtualenv <your-env>
+    source <your-env>/bin/activate
+    <your-env>/bin/pip install google-cloud-bigtable
 
-.. |pypi| image:: https://img.shields.io/pypi/v/google-cloud-bigtable.svg
-   :target: https://pypi.org/project/google-cloud-bigtable/
-.. |versions| image:: https://img.shields.io/pypi/pyversions/google-cloud-bigtable.svg
-   :target: https://pypi.org/project/google-cloud-bigtable/
+
+Windows
+^^^^^^^
+
+.. code-block:: console
+
+    pip install virtualenv
+    virtualenv <your-env>
+    <your-env>\Scripts\activate
+    <your-env>\Scripts\pip.exe install google-cloud-bigtable
+
+Next Steps
+~~~~~~~~~~
+
+-  Read the `Client Library Documentation`_ for Cloud Bigtable API
+   API to see other available methods on the client.
```

Comment:
Please remove "API" at the beginning of line 76, since the word is already at the end of like 75.